content/_data/plugins.json

```diff
@@ -929,6 +929,20 @@
           ],
           "badge": "community"
         },
+        {
+          "name": "Visual Regression Diff",
+          "description": "Perform visual regression test with a nice GUI as help. 💅 Supports both e2e and component-testing",
```

Comment:
```suggestion
          "description": "Perform visual regression test with a nice GUI as help. 💅 Supports both E2E and Component Testing",
```

Reply:
Hey @lmiller1990! Sure, it is a bit nicer

Reply:
added: 3b178e95e6af4a8213e8b7c3ce4ba2220edd213e